src/js/containers/covid19/OverviewContainer.jsx

```diff
@@ -38,11 +39,28 @@ const OverviewContainer = () => {
             window.removeEventListener('resize', handleWindowResize);
         };
     }, []);
-
     return (
         <div ref={divReference} className="body__content overview">
             <DateNote />
-            <AmountsVisualization overviewData={overviewData} width={visualizationsWidth} />
+            <AmountsVisualization
+                overviewData={{
+                    all: { ...overviewData },
```

Comment:
this is the same thing as overviewData.

```suggestion
                    all: overviewData,
```